Inc/HALALMock/Services/ADC/ADC.hpp

```diff
@@ -22,24 +23,31 @@ using std::string;
 #define LPTIM3_PERIOD 6875
 
 #define ADC_MAX_VOLTAGE 3.3
-#define MAX_12BIT 4095.0
-#define MAX_16BIT 65535.0
+#define MAX_16BIT 65535.0	// 2^16 - 1
+#define MAX_14BIT 16383.0	// 2^14 - 1
+#define MAX_12BIT 4095.0	// 2^12 - 1
+#define MAX_10BIT 1023.0	// 2^10 - 1
```

Comment:
This might be clearer:
```suggestion
#define MAX_16BIT 0b1111111111111111
#define MAX_14BIT 0b0011111111111111
#define MAX_12BIT 0b0000111111111111
#define MAX_10BIT 0b0000001111111111
```
Or by leaving the compiler to do the heavy lifting:
```suggestion
#define MAX_16BIT ((1<<16) - 1)
#define MAX_14BIT ((1<<14) - 1)
#define MAX_12BIT ((1<<12) - 1)
#define MAX_10BIT ((1<<10) - 1)
```